.gitignore

```diff
@@ -63,4 +63,5 @@ htmlcov/
 
 # CRDs used in the FV tests, pulled from libcalico.
 fv/infrastructure/crds.yaml
```

Comment:
Should this entry for crds.yaml be removed now?

Reply:
I left it because you might switch between new / old versions of the repo that use the old file.